text/0111-auto-resource-detection.md

```diff
@@ -0,0 +1,110 @@
+# Automatic Resource Detection
+
+Describes a mechanism to support auto-detection of resource information.
+
+## Motivation
+
+Resource information, i.e. attributes associated with the entity producing telemetry, can currently be supplied to tracer and meter providers or appended in custom exporters. In addition to this, it would be useful to have a mechanism to automatically detect resource information from the host (e.g. from an environment variable or from aws, gcp, etc metadata) and apply this to traces and metrics. This will in many cases prevent users from having to manually configure resource information.
```

Comment:
> and apply this to traces and metrics

And logs.

Reply:
Update all similar reference in this doc to "all kinds of telemetry"